analytics/views/support.py

```diff
@@ -179,20 +209,17 @@ def support(request: HttpRequest) -> HttpResponse:
                 context[
                     "success_message"
                 ] = f"Billing method of {realm.string_id} updated to charge automatically."
-        elif request.POST.get("sponsorship_pending", None) is not None:
-            sponsorship_pending = request.POST.get("sponsorship_pending")
-            if sponsorship_pending == "true":
+        elif sponsorship_pending is not None:
+            if sponsorship_pending:
                 update_sponsorship_status(realm, True, acting_user=acting_user)
                 context["success_message"] = f"{realm.string_id} marked as pending sponsorship."
-            elif sponsorship_pending == "false":
+            else:
                 update_sponsorship_status(realm, False, acting_user=acting_user)
                 context["success_message"] = f"{realm.string_id} is no longer pending sponsorship."
-        elif request.POST.get("approve_sponsorship") is not None:
-            if request.POST.get("approve_sponsorship") == "approve_sponsorship":
-                approve_sponsorship(realm, acting_user=acting_user)
-                context["success_message"] = f"Sponsorship approved for {realm.string_id}"
-        elif request.POST.get("downgrade_method", None) is not None:
-            downgrade_method = request.POST.get("downgrade_method")
+        elif approve_sponsorship:
```

Comment:
I think we should use `approve_sponsorship is True`.

Reply:
I don't see why that would be better than as written; can you explain?

Reply:
hmm. I realized it's not necessary since the value could be only `True` or `False` or `None`